ci-operator/jobs/openshift/aws-account-operator/openshift-aws-account-operator-master-presubmits.yaml

```diff
@@ -130,9 +134,15 @@ presubmits:
           requests:
             cpu: 10m
         volumeMounts:
+        - mountPath: /secrets/ci-pull-credentials
+          name: ci-pull-credentials
+          readOnly: true
         - mountPath: /secrets/gcs
           name: gcs-credentials
           readOnly: true
+        - mountPath: /secrets/hive-hive-credentials
+          name: hive-hive-credentials
```

Comment:
Why not just hive-credentials?

Reply:
This secret is part of the default secrets generated as a part of https://github.com/openshift/release/blob/889ee97c26570aa6860fdeb24acfe5509d040c88/core-services/ci-secret-bootstrap
secret name present here:- https://github.com/openshift/release/blob/889ee97c26570aa6860fdeb24acfe5509d040c88/core-services/ci-secret-bootstrap/_config.yaml#L1967

`*-presubmits.yaml` is an auto-generated yaml using [openshift-aws-account-operator-master.yaml file](https://github.com/openshift/release/pull/30474/files/4e00249352f6622d783858424f5175508ddf37c1#diff-bc5eb24b9029beee2b5ad639283be862b11323d209cbfa811e2896e0fcc1b28b)